docs/roadmap.md

```diff
@@ -0,0 +1,102 @@
+## Overview
+
+This is our public roadmap that outlines the high level direction we are working towards, namely [Themes](#themes). We update this document when our priorities change: security and stability is our top priority.
+
+[See our latest list of activities »](https://github.com/orgs/awslabs/projects/76/views/2){target="_blank"}
+
+## Themes
```

Comment:
(non-blocking)
Is Themes the right term here?

Some ideas: "Upcoming release", "High-priority features"

Reply:
Not all of them are features: i.e. improving end-to-end tests could be an area we might be working on, but it doesn't result in a feature.

And I'm a bit reticent with using "upcoming release" as it could generate the wrong expectations. This is somewhat related to the point above, but also to the fact that we don't want to commit to a specific timeline, at least for now. For instance, between now and when we finally release Idempotency, there might be many intermediate releases.